backend/db/sqliteschema/migrations.go

```diff
@@ -219,4 +219,66 @@ var migrations = []string{
 		-- The balance in satoshis
 		balance INTEGER DEFAULT 0
 	);`,
+
+	// Stores sites that user has manually added
+	`CREATE TABLE sites (
+		-- Site unique identification. The hostname of the site with protocol https://example.com
+		hostname TEXT PRIMARY KEY,
+		-- The role we play in the site ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL DEFAULT 0,
+		-- Address of the LND node backing up this wallet. In case lndhub, this will be the 
+		-- P2P addresses to connect to that site in the format of multiaddresses. Space separated.
+		addresses TEXT NOT NULL,
+		-- The account ID of the site. We need a previous connection to the site so the 
+		-- actual account is inserted in the accounts table when handshake.
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Table that stores all the tokens not yet redeemed inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE unredeemed_invite_links (
+		-- Unique token identification. Random 8 char words
+		token TEXT PRIMARY KEY,
+		-- The role the token will allow ROLE_UNSPECIFIED = 0 OWNER = 1 EDITOR = 2
+		role INT NOT NULL DEFAULT 2,
+		-- Timestamp since the token will no longer be eligible to be redeemed. Seconds since  Jan 1, 1970
+		expiration_time INTEGER NOT NULL CHECK (expiration_time > 0)
+	) WITHOUT ROWID;`,
+
+	// Table that stores the role each account has inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE redeemed_invite_links (
+		-- The account id that has been linked to a role on this site
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL PRIMARY KEY,
```

Comment:
@burdiyan it's better to link to an existing account id rather than just creating an unreferenced one here, right? Doing it with the reference would require having the account stored before adding a record in this table. Looking in the code, the peer account gets written in the db when performing the handshake when two nodes connect. Since connecting to the site is a required step before calling any remote function (via proxying) on it, we can guarantee that we are going to have the account id already in the `accounts` table, right?

Reply:
Yes, I think we should keep the data normalized unless there's reason not to.